sheep/http/http.c

```diff
@@ -437,7 +437,8 @@ int http_init(const char *options)
 	if (!sys->http_wqueue)
 		return -1;
 
-	FCGX_Init();
```

Comment:
Please separate it to another commit.

cc @RobinDong , it is related to http support.